numpy/distutils/fcompiler/ibm.py

```diff
@@ -35,9 +36,13 @@ def get_version(self,*args,**kwds):
             lslpp = find_executable('lslpp')
             xlf = find_executable('xlf')
             if os.path.exists(xlf) and os.path.exists(lslpp):
-                s, o = exec_command(lslpp + ' -Lc xlfcmp')
-                m = re.search(r'xlfcmp:(?P<version>\d+([.]\d+)+)', o)
-                if m: version = m.group('version')
+                try:
+                    o = subprocess.check_output([lslpp, '-Lc', 'xlfcmp'])
```

Comment:
This might be broken in the same way as #12831.

Reply:
I'm fine with reversing my attempt at removing `exec_command()` if that's the overall preference. After all, we didn't get any feedback from the original issue after the removal. It does seem preferable to avoid it at some point though, as Pauli noted.